governance/roles.md

```diff
@@ -20,60 +20,60 @@ All members are identified in the TAG [README](/README.md), with annotations
 where they hold an additional role.
 
 Members fulfilling any Roles in Security TAG are responsible for understanding
-and abiding the by the [governance](./) and policies defined in this group.
-This commitment and execution of understanding includes not only commits to
-the repo, but also to any approvals or direction required by their Role.
+and abiding the by the [governance](./) and policies defined in this group. This
+commitment and execution of understanding includes not only commits to the repo,
+but also to any approvals or direction required by their Role.
 
 **Write access to the repo**: Where applicable, the following roles define
 specific areas of the repo or actions on issues where changes require write
-access. In any case, governance is not enforced by [access permissions](github.md),
-but rather by members who are expected to thoughtfully consider their actions
-to support the group.
+access. In any case, governance is not enforced by [access
+permissions](github.md), but rather by members who are expected to thoughtfully
+consider their actions to support the group.
 
 ## Role of members
 
 * The primary role of a member is to contribute expertise to the group.
-* To add yourself as a member, submit a Pull Request (PR) adding yourself
-to the list of members.
+* To add yourself as a member, submit a Pull Request (PR) adding yourself to the
+  list of members.
 
 ### Maintaining active members
 
 * Members *SHOULD* remain active and responsive in their Roles.
 * Members taking an extended leave of 1 or more months *SHOULD* coordinate with
-other members to ensure the role is adequately staffed during the leave.
+  other members to ensure the role is adequately staffed during the leave.
 * Members going on leave for 1-3 months *MAY* work with other members to
-identify a temporary replacement.
-* Members of a role *SHOULD* remove any other members that have not
-communicated a leave of absence and either cannot be reached for more than 1
-month or are not fulfilling their documented responsibilities for more than 1
-month. This may be done through a super-majority vote of members, or if there
-are not enough *active* members to get a super-majority of votes cast, then
-removal may occur through a super-majority vote of the Chairs.
+  identify a temporary replacement.
+* Members of a role *SHOULD* remove any other members that have not communicated
+  a leave of absence and either cannot be reached for more than 1 month or are
+  not fulfilling their documented responsibilities for more than 1 month. This
+  may be done through a super-majority vote of members, or if there are not
+  enough *active* members to get a super-majority of votes cast, then removal
+  may occur through a super-majority vote of the Chairs.
```

Comment:
it's not quite clear here whether this is refering to our generic member list of members of a PROJECT role. i.e. removing from README or removing from issue/PR.

I don't think that there should be removal of folks from the README (but perhaps an active/inactive bit), since some that have made big contributions in the past but are no longer active should still be recognized as members.

Reply:
reading this over again, i think the Role specific clarifies this